Source/Core/Common/CMakeLists.txt

```diff
@@ -69,6 +69,14 @@ if (APPLE)
     ${COREFOUNDATION_LIBRARY}
     ${IOK_LIBRARY}
   )
+elseif(WIN32)
+  target_link_libraries(common
+  PRIVATE
+    kernel32
```

Comment:
Extra whitespace at the start (or, I guess more likely, all following lines need one more space for correct indent level)